rhombus/private/annotation.rkt

```diff
@@ -864,6 +865,31 @@
           #'arg-info.static-infos)
          #'tail)]))))
 
+(define-annotation-syntax satisfying
+  (annotation-prefix-operator
+   (annot-quote satisfying)
+   '((default . stronger))
+   'macro
+   (lambda (stx)
+     (syntax-parse stx
+       [(_ (_::parens pred-g) . tail)
+        (values
+         (annotation-predicate-form
+          ;; use `((lambda ....) ....)` to avoid inferred name
+          #'((lambda (pred)
```

Comment:
I guess `(lambda (pred) ...)` is inlined here to encourage optimization when `pred` is obviously a procedure of arity 1?

Reply:
Yes, that was the intention.